virttest/qemu_vm.py

```diff
@@ -1205,6 +1205,9 @@ def add_boot(devices, boot_order, boot_once, boot_menu, boot_strict):
                 cmd += " %s" % ",".join(options)
             else:
                 cmd = ""
+            if params.get("seabios_reboottimeout"):
```

Comment:
better use params["seabios_reboottimeout"]

Reply:
@jingzhao84  Thanks, I updated this part of code.